src/main/java/org/datadog/jenkins/plugins/datadog/traces/DatadogTraceBuildLogic.java

```diff
@@ -110,7 +111,6 @@ public void finishBuildTrace(final BuildData buildData, final Run<?,?> run) {
         buildSpan.setServiceName(DatadogUtilities.getDatadogGlobalDescriptor().getCiInstanceName());
         buildSpan.setType("ci");
         buildSpan.putMeta(CITags.CI_PROVIDER_NAME, "jenkins");
-        buildSpan.putMeta(CITags.LANGUAGE_TAG_KEY, "");
```

Comment:
unrelated but this tag is not needed now. It was reported when we used the APM Java tracer to report the traces